bridges/pangoro-goerli/bridge/src/service/header_relay/execution_layer_update.rs

```diff
@@ -124,14 +124,14 @@ impl ExecutionLayer {
                 .await?;
```

Comment:
Bad error message 
                    &self.pangoro_client.private_key.ok_or_else(|| {
                        BridgerError::Custom("Failed to get log_bloom from block".into())
                    })?,